go/vt/mysqlctl/schema.go

```diff
@@ -254,7 +255,7 @@ func (mysqld *Mysqld) normalizedSchema(ctx context.Context, dbName, tableName, t
 // ResolveTables returns a list of actual tables+views matching a list
 // of regexps
 func ResolveTables(ctx context.Context, mysqld MysqlDaemon, dbName string, tables []string) ([]string, error) {
-	sd, err := mysqld.GetSchema(ctx, dbName, tables, nil, true)
+	sd, err := mysqld.GetSchema(ctx, dbName, tables, nil, true, false)
```

Comment:
I think this can be `true` here as the columns information is not used in this function? I think this applies in more places than just here?

Reply:
Not sure? This function is called by vreplication here:

https://github.com/vitessio/vitess/blob/bc2f459cd175e5f52d97f1b52244019b4ae03e3b/go/vt/vttablet/tabletmanager/vreplication/controller.go#L229-L235

And this goes deep, but normally vreplication is interested in column/field information; I'd need to dig to verify whether it does or does not, but it feels like a risk.

Reply:
@shlomi-noach `ResolveTables` returns a list of the tables, it gets the `Name` from each entry in `TableDefinitions` so that wouldn't include any column information? And that then is returned, so from what I see in the code all the rest of the data from the schema is thrown away?

Reply:
@dbussink you're right. I see it now. Changed to `true` with comment.